app/assets/javascripts/models/beta/iteration.js

```diff
@@ -134,14 +175,13 @@ const fillRemainingPoints = (sprints, storyPoints, index, velocity, previousFill
   }
 };
 
-const createFirstSprint = (sprints, project, initialSprintNumber, velocity) => {
-  sprints.push(createSprint(
+const createFirstSprint = (sprints, project, initialSprintNumber, velocity) =>
```

Comment:
Are you using `sprints` parameter?

Reply:
Nop, before of my change it was used, but now i will remove. thanks!